src/Ledger/Ratify.lagda

```diff
@@ -362,25 +362,29 @@ activeVotingStake : ℙ VDeleg → StakeDistrs → (VDeleg ⇀ Vote) → Coin
 activeVotingStake cc dists votes =
   Σᵐᵛ[ x  ← getStakeDist DRep cc dists ∣ dom votes ᶜ ᶠᵐ ] x
 
-accepted' : RatifyEnv → EnactState → GovActionState → Set
-accepted' Γ (record { cc = cc , _ ; pparams = pparams , _ }) gs =
-  acceptedBy CC ∧ acceptedBy DRep ∧ acceptedBy SPO ∧ meetsMinAVS
+module _ (Γ : RatifyEnv) (es : EnactState) (gs : GovActionState)
```

Comment:
One issue with putting this in a module is that we want to avoid module as much as we can in the PDF, which is why the `where` block was a good solution. The issue is of course that we now need `acceptedBy` outside of the scope of `accepted`. I think in that case it might be better to split this into two functions at the top level at the cost of repeating the telescope of arguments twice.

Reply:
Can do, but not that it's not just the telescope, but also `votes'`, `cc'` and `redStakeDistr`.

Reply:
Ahh, you're right. Well, I have some good news: I'm almost certain we won't have `minimumAVS` in the final version of the spec, so I think it's safe to remove it, and then those definitions can just go in `acceptedBy`.